### DISCUSSION TITLE
Connector Builder: Add support for multiple base URL'S

### DISCUSSION THREAD

**Discussion by btkcodedev** [airbytehq/airbyte#44373] [Connector Builder] [answered]
Currently, a single base URL is supported by the connector builder, please make it support multiple base URL
CC: @lmossman 
Ref Source with multiple bases: 
Datadog Source: https://github.com/airbytehq/airbyte/pull/44371

Error from the builder:
![image](https://github.com/user-attachments/assets/58c29e0c-6a95-47c9-a1ea-76a1ef8bd1da)

**Comment by brianbolt** [comment]
This would be a good feature. I am currently dealing with an api which had a search domain and a data domain.  I want to be able to search and then call the data domain with the resulting ids as a substream but the low code solution doesn't work.  I tried to "hack" the low code solution by trying https:// as the base url and then supply the full domain as the path but the way the low code solution tries to assemble the URLs does not allow for this.

**Comment by plovegro** [comment, marked as answer]
Just revisiting this as i have a hack.
For the base URL at the top level i am using **http://**
Then for each stream i use the full address - ie. **https://pro-api.coinmarketcap.com/v1/cryptocurrency/listings/latest**
And it works! I tried different combos, but they didnt work. This method did! 
@btkcodedev @brianbolt
